lib/matplotlib/axes/_axes.py

```diff
@@ -4792,6 +4792,8 @@ def _pcolorargs(funcname, *args, **kw):
 
         if len(args) == 3:
             X, Y, C = args
+            X = np.asarray(X)
+            Y = np.asarray(Y)
```

Comment:
Are masked arrays supported for coordinates in the pcolor family?

On Wed, Jul 1, 2015 at 10:59 AM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/4573#discussion_r33686784>:
>
> > @@ -4792,6 +4792,8 @@ def _pcolorargs(funcname, *args, **kw):
> >
> >          if len(args) == 3:
> >              X, Y, C = args
> > +            X = np.asarray(X)
> > +            Y = np.asarray(Y)
>
> and masked arrays.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4573/files#r33686784>.
>